### DISCUSSION TITLE
Number of autodiscoved repositories no longer printed on INFO in 37.x

### DISCUSSION THREAD

**Discussion by szpak** [renovatebot/renovate#26444] [Suggest an Idea]
### Tell us more.

Up to Renovate 36.x, with the autodiscovered repositories their number was also printed:

```
 INFO: Autodiscovered repositories
       "length": 205,
       "repositories": [
         ...
       ]
```

In Renovate 37.x (checked with 37.107.0, I've just migrated to), the "length" is missing:

```
 INFO: Autodiscovered repositories
       "repositories": [
         ...
       ]
```

Of course, I can call jq at the end and calculate and print the number, but it was a quick way to see number of (still growing) "covered by Renovate" repositories visible at the beginning of the CI longs in every job.

I wonder, if it was removed deliberately for some reasons, or it "just happened". In the other case, maybe it would be worth to restore printing that number in the INFO logs?

**Comment by rarkins** [comment]
It's ok to restore it - PR welcome

**Reply by szpak** [reply]
Created https://github.com/renovatebot/renovate/pull/26458/

**Comment by szpak** [comment]
Released in [37.115.0](https://github.com/renovatebot/renovate/releases/tag/37.115.0).
